commit d4f42b71e816b19dd44b373e1006ca64da75c192
Author: Ran.Park <ranpark@foxmail.com>
Date:   Wed Feb 14 19:40:08 2024 +0800

    Fixed case issue with 'fault-injection' in documentation
    
    In the 'fault-injection' subdirectory, the first letter F
    is capitalized, whereas in index.rst f is lowercase, but in
    index.rst all other elements in the same column are capitalized.
    
    Signed-off-by: "Ran.Park" <ranpark@foxmail.com>
    Signed-off-by: Jonathan Corbet <corbet@lwn.net>
    Link: https://lore.kernel.org/r/tencent_3EA07E65C43816C2A8402DC655CF98916B06@qq.com

diff --git a/Documentation/fault-injection/index.rst b/Documentation/fault-injection/index.rst
index 8408a8a91b34..a6ea1d190222 100644
--- a/Documentation/fault-injection/index.rst
+++ b/Documentation/fault-injection/index.rst
@@ -1,7 +1,7 @@
 .. SPDX-License-Identifier: GPL-2.0
 
 ===============
-fault-injection
+Fault-injection
 ===============
 
 .. toctree::